src/libsystemd-network/sd-radv.c

```diff
@@ -590,6 +598,58 @@ _public_ int sd_radv_set_rdnss(sd_radv *ra, uint32_t lifetime,
         return 0;
 }
 
+_public_ int sd_radv_set_dnssl(sd_radv *ra, uint32_t lifetime,
+                               const char **search_list) {
```

Comment:
because of the non-obvious "const" semantics when dealing with char** arrays we usually don't use "const" at all of them... As you noticed you have to cast in some cases otherwise, which is hardly beneficial...